src/Robo/Commands/Setup/SettingsCommand.php

```diff
@@ -217,15 +217,16 @@ public function gitHooks() {
   protected function installGitHook($hook) {
     if ($this->getConfigValue('git.hooks.' . $hook)) {
       $this->say("Installing $hook git hook...");
-      $source = $this->getConfigValue('git.hooks.' . $hook) . "/$hook";
-      $dest = $this->getConfigValue('repo.root') . "/.git/hooks/$hook";
+      $bltHook = $this->getConfigValue('git.hooks.' . $hook) . "/$hook";
```

Comment:
This isn't necessarily a BLT hook, it could be custom. Why not name it something like `$source_hook`?

Reply:
hmm, it's a BLT hook in the sense that it's the hook that ships with BLT in it's vendor directory:
```
  hooks:
    pre-commit: ${blt.root}/scripts/git-hooks
    commit-msg: ${blt.root}/scripts/git-hooks
```

Reply:
I can change it to `$hook_source` if you prefer, was thinking it was essentially a "blt provided hook"